tests/backends/tests.py

```diff
@@ -745,6 +745,19 @@ def test_cursor_contextmanager_closing(self):
             self.assertIsInstance(cursor, CursorWrapper)
         self.assertTrue(cursor.closed)
 
+    @skipUnlessDBFeature('test_db_allows_multiple_connections')
```

Comment:
> For the sqlite3 case the connection will never close (it can't) so the test will fail. The bug can only be reproduced with postgresql oracle and mysql so testing sqlite is pointless to me.

Keep in mind that while the default test settings use an in memory SQLite database running the suite with a file based SQLite database is a valid setup that should be accounted for.